samples_project/Assets/SampleViewer/Samples/ThirdPerson/readme.md

```diff
@@ -0,0 +1,33 @@
+# Explore with a Third Person Character
```

Comment:
```suggestion
# Explore with a Third Person Character Controller
```
The character isn't third person, the camera/controller is. Not sure about this title, I'm open to new ideas.